### DISCUSSION TITLE
[27 giugno 2023] Esercizio 2

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#252] [Esami]
Un circuito combinatorio riceve $\Large 4$ bit in ingresso $\Large abcd$ e produce in uscita $\Large 4$ bit che rappresentano il valore $\Large Y= y_3y_2y_1y_0$ , in complemento a $\Large 2$, dove $\Large Y=A+B$ con $\Large A$ numero naturale dato dai bit $\Large ab$ e $\Large B$ numero in complemento a $\Large 2$ rappresentato dai bit $\Large bcd$.

1. Scrivere la tavola di verità.

2. Realizzare $\Large Y$ con una PLA con il minimo numero di righe.

3. Realizzare $\Large y_3$ con un MUX 4-a-1.

4. Realizzare $\Large y_2$ con sole porte NAND.

**Comment by M1SC0** [comment]
![image](https://github.com/user-attachments/assets/48334650-e473-41ac-a27d-6ef7c82a7dd2)
![image](https://github.com/user-attachments/assets/2a11a657-f025-41ca-b308-6419bfecdf55)
![image](https://github.com/user-attachments/assets/b06a6ccf-1ca8-4636-adbd-741eb60dfbe5)
![image](https://github.com/user-attachments/assets/8fad4789-c7c3-4209-9f2a-f65b38fdcfe6)
![image](https://github.com/user-attachments/assets/053bf8b8-871d-41fb-b55b-06e442ecdb65)
